src/components/app/Home.css

```diff
@@ -85,7 +85,8 @@
   line-height: 1.5;
   text-align: center;
   text-decoration: none;
-  transition: background-color 300ms;
+  transition: background-color 250ms var(--animation-curve),
+    color 250ms var(--animation-curve);
```

Comment:
I tried to reduce to 150ms but it was really too fast IMO.

Reply:
Looks good to me this way 👍